lib/Transforms/Scalar/UnifyAssumes.cc

```diff
@@ -135,6 +181,34 @@ bool UnifyAssumesPass::runOnFunction(Function &F) {
   return false;
 }
 
+void UnifyAssumesPass::processAssertInst(CallInst &CI, AllocaInst &flag) {
+  BasicBlock *bb = CI.getParent();
+  Module *M = bb->getParent()->getParent();
+  assert(bb);
+  assert(M);
+
+  auto &SBI = getAnalysis<seahorn::SeaBuiltinsInfoWrapperPass>().getSBI();
```

Comment:
make `SBI` a field and instantiate it during `runOnFunction`

Reply:
passing it explicitly to callee since best practice with class field would suggest setting it at object creation time.

Reply:
No, this is an LLVM idiom. `runOnFunction` is essentially a creation method, analyses are extracted at that time and stored in fields.